cryodrgn/commands/eval_vol.py

```diff
@@ -232,7 +235,7 @@ def main(args):
             vol = vol[::-1]
         if args.invert:
             vol *= -1
-        mrc.write(args.o, vol.astype(np.float32), Apix=args.Apix)
+        MRCFile.write(args.o, np.array(vol).astype(np.float32), Apix=args.Apix)
```

Comment:
I introduced an `MRCFile` class in the hope of future integration with `MRCHeader` class. Currently its just a class with a single `classmethod` though, so equivalent to the already existing free-floating `mrc.write` function. I can revert this back in case it helps simplify parsing this PR.

Reply:
I don't feel strongly either way.